src/components/LoginPage/LoginPageComponent.js

```diff
@@ -30,14 +30,10 @@ const H3 = styled.h3`
   color: ${COLORS.white};
 `;
 
-// 2% - indent for image positioning
-// 92% - image height without vertical indention
-// 4% - vertical indention
-// 0.75% - horizontal indention
-const GoogleLoginButton = styled.button`
+const buttonIn = css`
```

Comment:
What does `buttonIn` mean? :)

Button in... what? :)

Reply:
Yeah, not the best of my decisions )  What about named it just Button? It is the common button style for this page